### PR TITLE
docs: some recent workarounds for recent problems

### PR DESCRIPTION
**Pull request by rgraber**
----

I've completed each of the following or determined they are not applicable:

- [ ] Made a plan to communicate any major developer interface changes (or N/A)

### CODE REVIEW

File: docs/troubleshoot_general_tips.rst [hunk 1]
```diff
@@ -214,3 +214,20 @@ Another error you may get if the code and the image are out of sync is sql or Dj
...
+
+CORS error from login_refresh in MFE
+------------------------------------
+If you see "Access to XMLHttpRequest at 'http://localhost:18000/login_refresh' from origin 'http://localhost:2000' has been blocked by CORS policy: Request header field x-xsrf-token is not allowed by Access-Control-Allow-Headers in preflight response" it usually means you don't have a valid session.
+
+The fix is to get a new auth session. You can
+
+1. First surf to http://localhost:18000 to restart your logged in http session.
+2. Log out and then back in.
+3. Clearing your cookies, then log back in.
```
Comment (timmc-edx) [inline docs/troubleshoot_general_tips.rst]: [nit] Tense match
```suggestion
3. Clear your cookies, then log back in.
```

File: docs/troubleshoot_general_tips.rst [hunk 2]
```diff
@@ -214,3 +214,20 @@ Another error you may get if the code and the image are out of sync is sql or Dj
...
+
+To fix, simply add a new subsection and publish. The act of publishing should reload the whole course correctly.
+
+CORS error from login_refresh in MFE
+------------------------------------
+If you see "Access to XMLHttpRequest at 'http://localhost:18000/login_refresh' from origin 'http://localhost:2000' has been blocked by CORS policy: Request header field x-xsrf-token is not allowed by Access-Control-Allow-Headers in preflight response" it usually means you don't have a valid session.
+
+The fix is to get a new auth session. You can
+
+1. First surf to http://localhost:18000 to restart your logged in http session.
```
Comment (timmc-edx) [inline docs/troubleshoot_general_tips.rst]: Are these three list items different alternatives, or steps in a sequence? I think it's supposed to be the former, but the "First" is throwing me off a bit.

**Review by timmc-edx — APPROVED** [review]

File: docs/troubleshoot_general_tips.rst [hunk 3]
```diff
@@ -214,3 +214,20 @@ Another error you may get if the code and the image are out of sync is sql or Dj
...
+To fix, simply add a new subsection and publish. The act of publishing should reload the whole course correctly.
+
+CORS error from login_refresh in MFE
+------------------------------------
+If you see "Access to XMLHttpRequest at 'http://localhost:18000/login_refresh' from origin 'http://localhost:2000' has been blocked by CORS policy: Request header field x-xsrf-token is not allowed by Access-Control-Allow-Headers in preflight response" it usually means you don't have a valid session.
+
+The fix is to get a new auth session. You can do any of the following:
+
+1. Before navigating to your MFE, go to http://localhost:18000 to restart your logged in http session.
+2. Log out and then back in.
```
Comment (robrap) [inline docs/troubleshoot_general_tips.rst:231]: Can we drop this step altogether? Why log out or log in right before we have them clear cookies?

File: docs/troubleshoot_general_tips.rst [hunk 4]
```diff
@@ -214,3 +214,20 @@ Another error you may get if the code and the image are out of sync is sql or Dj
...
+3. Clear your cookies, then log back in.
```
Comment (robrap) [inline docs/troubleshoot_general_tips.rst:232]: Nits:
1. Should we separate "Log in" to be its own step 4?
2. Why say "log back in" vs "log in", if the problem might be that they were never logged in?

**Pull request merged**